cmd/alertmanager/main.go

```diff
@@ -411,7 +411,9 @@ func main() {
 		for {
 			select {
 			case <-hup:
-				reload()
+				if err := reload(); err != nil {
+					level.Error(logger).Log("msg", fmt.Sprintf("failed to reload config: %v", err))
```

Comment:
This would introduce double error logging because the `reload()` function already logs this itself: https://github.com/prometheus/alertmanager/blob/0e361ccd6b0ae8704a541c5909f24e585400edb5/cmd/alertmanager/main.go#L325